src/_data/sites/montevil.json

```diff
@@ -0,0 +1,8 @@
+{
+    "url": "https://montevil.org/",
+    "name": "montevil.org",
+    "description": "Maël Montévil research website",
+    "twitter": "mmontevil",
+    "authoredBy": ["Maël Montévil"],
```

Comment:
Ah, per the docs (or [new issue template](https://github.com/11ty/11ty-website/issues/new?assignees=&labels=new-site%21&template=built-with-eleventy.md&title=I+built+a+site+with+Eleventy%21)):

> `authoredBy`: An array of Twitter usernames of the site’s authors. Supplements the `twitter` entry. (Optional)

So I think we can omit this field since your Twitter handle is already defined above.